draft-ietf-cdni-ci-triggers-rfc8007bis.xml

```diff
@@ -2028,1226 +1972,1147 @@ uCDN-A                         dCDN-B                         dCDN-C
-   By default, each record in the list is assumed to represent an object that does not require additional processing.
+   Description: The CDN PID of the CDN where the error occurred. The "cdn-id" property is used
+                by the originating uCDN or by the propagating dCDN in order to distinguish in which CDN the 
+                error occurred.
    <vspace blankLines="1"/>
-   Value: ObjectListType (see <xref target="object-list"/>).
-          <vspace blankLines="1"/>
-   Mandatory: Yes.
+   Value:  A non-empty JSON string, where the string is a CDN PID as defined in
+                 <xref target="loop-detection-and-prevention"/>
+   <vspace blankLines="1"/>
+   Mandatory: Yes. dCDN may use its own CDN PID if it does not want to expose the CDN PIDs of dCDNs.
    </t>
+
    </list></t>
    </list></t>
 
-   <section title="ObjectList Type" anchor="object-list-type">
-   <t>
-      ObjectListType objects are used to specify the registered type of ObjectList objects
-      (see <xref target="IANA.CDNI.ObjectListTypeReg"/>), used in Trigger Spec, 
-      Trigger Status and Error.v2 Description objects.
+   <t><list style="empty" hangIndent="3">
+   <t><list style="hanging" hangIndent="3"><t hangText="Name: objects">
+   <vspace blankLines="1"/>
+   Description: List of objects that failed to be processed during trigger execution.
+   <vspace blankLines="1"/>
+   Value:   An array of <xref target="object-list">ObjectList</xref> objects. dCDN SHOULD provide the list of objects that it failed to process during trigger execution with <xref target="objectlist-spec"/>, provided that dCDN advertised <xref target="extended-status-capability-object">support for extended status</xref>.
+   <vspace blankLines="1"/>
+   Mandatory: No. An empty array is allowed and is equivalent to omitting "objects" from the Error.v2 Description.
    </t>
+   </list></t>
+   </list></t>
+
+
+
+          <t>Example of a JSON-serialized Error.v2 Description object reporting a malformed object list :
+          </t>
+
+          <sourcecode type="json" name="Error.v2 Description">
+<![CDATA[
+{
+  "error": "econtent",
+  "description": "Failed to parse HLS object list",
+  "specs": [{
+    "trigger-subject": "content",
+    "generic-trigger-spec-type": "content-objectlist",
+    "generic-trigger-spec-value": {
+      "objects": [{
+        "href": "https://www.example.com/hls/title/index.m3u8",
+        "type": "hls"
+      }]
+   }}
+  ],
+  "objects": [{ 
+    "href": "https://www.example.com/hls/title/index.m3u8", 
+    "type": "hls"
+  }],
+  "cdn": "AS64500:0"
+}
+         ]]></sourcecode>
+
+          <t>Example of a JSON-serialized Error.v2 Description object reporting an unsupported extension object:
+          </t>
+
+<sourcecode type="json" name="Error.v2 Description unsupported extension"><![CDATA[
+{
+  "errors": [{
+    "error": "eextension",
+    "description": "unrecognized extension <type>",
```

Comment:
Nit: Now that we have filled out the extensions section (which just had placeholders before), and there is only one extension listed, we should probably replace <type> with "location-policy".

Reply:
ACK